torch/_logging/_internal.py

```diff
@@ -118,6 +118,18 @@ def clear(self):
 log_registry = LogRegistry()
 log_state = LogState()
 
+# Groups of logging configurations that can be used with `set_logs`.
```

Comment:
Can you put this in registrations and make a small API for it?
ie: `register_log_group(<alias>, <set of aliases to turn on>)`

I think this could be useful for people to create their own logging groups in the future